src/http/api/views/account.posts.view.ts

```diff
@@ -112,6 +110,11 @@ export class AccountPostsView {
         );
     }
 
+    /**
+     * Get posts by account
+     *
+     * @deprecated Use getPostsFromOutbox instead
+     */
```

Comment:
Will remove this in the next PR. On removing this, Github was creating too many diff between this and the new query.